libs/blocks/ost/ost.js

```diff
@@ -1,24 +1,77 @@
-import { loadScript, loadStyle } from '../../utils/utils.js';
+import { getConfig, getLocale, loadScript, loadStyle } from '../../utils/utils.js';
+import { ENV_PROD, getTacocatEnv, getTacocatMetadata } from '../merch/merch.js';
 
 const IMS_COMMERCE_CLIENT_ID = 'aos_milo_commerce';
 const IMS_PROD_URL = 'https://auth.services.adobe.com/imslib/imslib.min.js';
 const OST_SCRIPT_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/1.10.0/index.js';
 const OST_STYLE_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/1.10.0/index.css';
 
-const ENVIRONMENT = 'PROD';
-const WCS_API_KEY = 'wcms-commerce-ims-ro-user-cc';
-const AOS_API_KEY = 'wcms-commerce-ims-user-prod';
-const CHECKOUT_CLIENT_ID = 'creative';
+export const AOS_API_KEY = 'wcms-commerce-ims-user-prod';
+export const CHECKOUT_CLIENT_ID = 'creative';
+export const WCS_API_KEY = 'wcms-commerce-ims-ro-user-cc';
 
 const searchParameters = new URLSearchParams(window.location.search);
-// this is only for testing PRs where test URLs are not supported by IMS.
+
+/*
+  This prevents IMS redirects to sign-in page and back
```

Comment:
I wonder if it's good idea to include this in a public repo?

Reply:
Very good point, @Axelcureno. Let's discuss this all together